### PR TITLE
feat(*): modularize grid

### PR DESCRIPTION
**Pull request by awinogradov**
ENB module for easy usage and dist. ENB module will be available in NPM after time when this PR will be accepted.
#106, #102, #101, #71, #99

### CODE REVIEW

File: .enb/make.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ var path = require('path'),
         files : require('enb-bem-techs/techs/files'),
         postcss : require('enb-postcss/techs/enb-postcss'),
         bemhtml : require('enb-bemxjst/techs/bemhtml'),
-        html : require('enb-bemxjst/techs/html-from-bemjson')
+        borschik : require('enb-borschik/techs/borschik'),
+        html : require('enb-bemxjst/techs/bemjson-to-html'),
+        grid : require('../engines/enb-grid/techs/enb-grid')
```
Comment (voischev) [inline .enb/make.js]: rename enb-bem-grid?

File: .enb/make.js [hunk 2]
```diff
@@ -19,7 +21,7 @@ module.exports = function(config) {
             [techs.levels, {
                 levels : [
                     { path : path.join('libs', 'bem-core',  'common.blocks'), check : false },
-                    { path : 'common.blocks', check : true },
+                    { path : path.join('libs', 'bem-components',  'common.blocks'), check : false },
```
Comment (zxqfox) [inline .enb/make.js]: Пробел лишний!

File: .enb/make.js (hunk 1 above)
Comment (awinogradov) [inline .enb/make.js]: Why? No variants) Always bem-grid;)

Comment (zxqfox) [inline .enb/make.js]: For consistency!

**Pull request merged**